### PR TITLE
add create base scaffold header

### PR DESCRIPTION
**Pull request by Soubpwnz**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  width: 20%;
+}
+
+.image {
+  position: relative;
+
+  width: 100%;
+  top: -20px;
+
```
Comment (vinnikukr) [inline assets/css/style.css:28]: This row redundant

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  width: 150px;
+  height: 150px;
+  border-radius: 50%;
+
+  margin: 10px auto;
+
+  outline-offset: 2px;
+  outline: 1px solid black;
+
+
```
Comment (vinnikukr) [inline assets/css/style.css:42]: This rows redundant

File: index.html [hunk 3]
```diff
@@ -1,15 +1,34 @@
...
+  <link rel="stylesheet" href="assets/css/style.css">
 </head>
 
 <body>
-  Hello world
+  <div class="container">
+
+    <!--my left sidebar-->
+    <aside class="left_sidebar">
+      <div class="image-vrap">
```
Comment (vinnikukr) [inline index.html:19]: You should choose one naming style: or - , or _

File: index.html [hunk 4]
```diff
@@ -1,15 +1,34 @@
...
+        <img class="image" src="./img/Photo.jpg" alt="Photo">
+      </div>
+    </aside>
+
+    <!--a bit about me-->
+    <div class="container-text">
+      <div class="about_me">
```
Comment (vinnikukr) [inline index.html:26]: FYI

**Review by vinnikukr — COMMENTED** [review]
Folder "img" should be in folder "assets"

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+.image {
+  position: relative;
+
+  width: 100%;
+  top: -20px;
+
+}
+
+.image-vrap {
```
Comment (dissolvent) [inline assets/css/style.css:31]:
```suggestion
.image-wrap {
```
I believe it's typo

File: index.html [hunk 6]
```diff
@@ -1,15 +1,34 @@
...
+
+  <link rel="stylesheet" href="assets/css/style.css">
 </head>
 
 <body>
-  Hello world
+  <div class="container">
+
+    <!--my left sidebar-->
+    <aside class="left_sidebar">
```
Comment (dissolvent) [inline index.html:18]: Not sure that ```aside``` is suitable here
[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)

File: index.html [hunk 7]
```diff
@@ -1,15 +1,34 @@
...
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>public-cv</title>
+
+  <link rel="stylesheet" href="assets/css/style.css">
 </head>
 
 <body>
-  Hello world
+  <div class="container">
```
Comment (dissolvent) [inline index.html:15]: I think you can use ```main``` tag here 
[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main)

**Review by dissolvent — APPROVED** [review]

**Pull request merged**